data/mods/Magiclysm/Spells/technomancer.json

```
@@ -559,7 +559,7 @@
     "type": "SPELL",
     "name": "X-ray Vision",
     "id": "x-ray",
-    "description": "You fire a cone of X-rays that magically allows you to see that area for a short time.",
+    "description": "You fire a cone of X-rays that magically allow you to see that area for a short time.",
```

```suggestion
    "description": "You fire a cone of X-rays, magically allowing you to see everything they pass through for a short time.",
```
This still sounds clunky, maybe something like this instead? (Admittedly I've never used this so idk if the description is accurate or not)

---

Summarizing Discord followup: the string with "allow" already exists elsewhere, so changing "allows" to "allow" shouldn't trigger any new translation effort.